### ISSUE TITLE
Switch Pro Controller seen but not registering any input

### ISSUE THREAD

**Issue by robross0606** [KeyPuncher/WiinUPro#28]
WiinUPro is seeing the controller as it shows up under devices.  However, after connection and going to the UI for the controller, it doesn't register any button presses at all and cannot be used in any game.  It shows up as DInput device on Dolphin but, again, not registering any button presses.  It works fine in Steam.

**Comment by robross0606** [comment]
I should also mention that I successfully paired multiple Wiimotes and nunchucks and all inputs are registering and usable in games.  Only the switch pro controller seems to have this issue.

**Comment by KeyPuncher** [comment]
Steam's implementation is really good, especially once you figure out how to do all the advanced stuff.

The Pro is connected via Bluetooth right? (I don't think I've added USB support but my memory isn't great). I need to implement communication directly with the Switch controllers to have more control instead of piggy backing off of the DInput it gives off.

**Comment by robross0606** [comment]
Yes, BetterJoyForCemu send to work okay but needed the special version that supports analog calibration.

**Comment by Wrexes** [comment]
Isn't the Switch Pro Controller natively supported by Windows and thus, most games ?
I remember just pairing it and using it straight away without any custom driver or anything.

**Comment by TheShadowGamer** [comment]
I'm also having this problem, but I am connected via USB. Has anyone found any solutions for this?
